AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -223,6 +223,20 @@ interface TestRunnerService {
             )
         }
     }
+    data class TestIdentifier(
+        /**
+         * The name of the class.
+         */
+        public val className: String? = null,
```

Comment:
Why are these nullable? It seems like we create these outselves and having TestIdentifier that has a null value for either className or name would mean we cannot use this identifier to determine what it points to. Lets make them nonnull.